[2025-03-13T04:58:29.658+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-13T04:58:29.743+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.merge_all_trips manual__2025-03-13T04:53:20.200428+00:00 [queued]>
[2025-03-13T04:58:29.780+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.merge_all_trips manual__2025-03-13T04:53:20.200428+00:00 [queued]>
[2025-03-13T04:58:29.782+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 2
[2025-03-13T04:58:29.809+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): merge_all_trips> on 2025-03-13 04:53:20.200428+00:00
[2025-03-13T04:58:29.819+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2732) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-13T04:58:29.823+0000] {standard_task_runner.py:72} INFO - Started process 2741 to run task
[2025-03-13T04:58:29.824+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'merge_all_trips', 'manual__2025-03-13T04:53:20.200428+00:00', '--job-id', '591', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpmyzsrhdj']
[2025-03-13T04:58:29.826+0000] {standard_task_runner.py:105} INFO - Job 591: Subtask merge_all_trips
[2025-03-13T04:58:29.894+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_gcs_dag.merge_all_trips manual__2025-03-13T04:53:20.200428+00:00 [running]> on host b0577ed9eb82
[2025-03-13T04:58:30.066+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_gcs_dag' AIRFLOW_CTX_TASK_ID='merge_all_trips' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T04:53:20.200428+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-13T04:53:20.200428+00:00'
[2025-03-13T04:58:30.068+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-13T04:58:30.069+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-13T04:58:30.069+0000] {logging_mixin.py:190} INFO - Current task name:merge_all_trips state:running start_date:2025-03-13 04:58:29.744231+00:00
[2025-03-13T04:58:30.070+0000] {logging_mixin.py:190} INFO - Dag name:data_ingestion_gcs_dag and current dag run status:running
[2025-03-13T04:58:30.070+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-13T04:58:47.334+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-13T04:58:47.350+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-13T04:58:47.351+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=merge_all_trips, run_id=manual__2025-03-13T04:53:20.200428+00:00, execution_date=20250313T045320, start_date=20250313T045829, end_date=20250313T045847
[2025-03-13T04:58:47.384+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-13T04:58:47.385+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-13T04:58:47.385+0000] {logging_mixin.py:190} INFO - Dag name:data_ingestion_gcs_dag queued_at:2025-03-13 04:53:20.246651+00:00
[2025-03-13T04:58:47.386+0000] {logging_mixin.py:190} INFO - Task hostname:b0577ed9eb82 operator:PythonOperator
[2025-03-13T04:58:47.405+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-13T04:58:47.438+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-13T04:58:47.441+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
